src/core_plugins/tile_map/public/kibana_map.js

```diff
@@ -100,7 +100,9 @@ export class KibanaMap extends EventEmitter {
       minZoom: options.minZoom,
       maxZoom: options.maxZoom,
       center: options.center ? options.center : [0, 0],
-      zoom: options.zoom ? options.zoom : 2
+      zoom: options.zoom ? options.zoom : 2,
+      renderer: L.canvas(),
```

Comment:
by using canvas now, I'd hoped we could have turned off this client-side filtering hack:

 https://github.com/thomasneirynck/kibana/blob/72f6b8a37fe76c9d799fc7af9f7dd2358789d987/src/core_plugins/tile_map/public/markers/scaled_circles.js#L30-L35
 https://github.com/thomasneirynck/kibana/blob/23d903746383c2fc3d55c0cf1075a686aa6cb0a9/src/core_plugins/tile_map/public/geohash_layer.js#L92-L92

However, when turning this off, it remains pretty jittery. so ill guess we should leave it in. :(